CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ The latest version is considered WIP and it is a subject of change. All other
 versions are frozen. To freeze current version all files are copied to a new
 version directory, and then changes are introduced.
 
+## [v3.2.4] WIP
```

Comment:
I think we agreed to not mark cloudconfig as WIP because it should be ready for release in any time.

Reply:
There is no way. Everything should be released every time.